r/PACKAGING.md

```diff
@@ -18,86 +18,71 @@
   under the License.
 -->
 
-# Packaging checklist for CRAN release
+# Packaging Checklist for CRAN Release
 
-For a high-level overview of the release process see the
-[Apache Arrow Release Management Guide](https://arrow.apache.org/docs/developers/release.html#post-release-tasks).
+For a high-level overview of the Arrow release process see the [Apache Arrow Release Management Guide](https://arrow.apache.org/docs/developers/release.html#post-release-tasks).
 
-## Before the release candidate is cut
+## Before the Arrow Release Candidate Is Created
 
 - [ ] [Create a GitHub issue](https://github.com/apache/arrow/issues/new/) entitled `[R] CRAN packaging checklist for version X.X.X` and copy this checklist to the issue.
 - [ ] Review deprecated functions to advance their deprecation status, including removing preprocessor directives that no longer apply (search for `ARROW_VERSION_MAJOR` in r/src).
 - [ ] Evaluate the status of any failing [nightly tests and nightly packaging builds](http://crossbow.voltrondata.com). These checks replicate most of the checks that CRAN runs, so we need them all to be passing or to understand that the failures may (though won't necessarily) result in a rejection from CRAN.
-- [ ] Check [current CRAN check results](https://cran.rstudio.org/web/checks/check_results_arrow.html)
+- [ ] Check [current CRAN check results](https://cran.rstudio.org/web/checks/check_results_arrow.html).
 - [ ] Ensure the contents of the README are accurate and up to date.
 - [ ] Run `urlchecker::url_check()` on the R directory at the release candidate.
   commit. Ignore any errors with badges as they will be removed in the CRAN release branch.
-- [ ] [Polish NEWS](https://style.tidyverse.org/news.html#news-release) but do **not** update version numbers (this is done automatically later). You can find commits by, for example, `git log --oneline <sha of last release>..HEAD | grep "\[R\]"`
-- [ ] Run preliminary reverse dependency checks using `archery docker run r-revdepcheck`.
-- [ ] For major releases, prepare tweet thread highlighting new features.
+- [ ] [Polish NEWS](https://style.tidyverse.org/news.html#news-release) but do **not** update version numbers (this is done automatically later). You can find commits by, for example, `git log --oneline <sha of last release>..HEAD | grep "\[R\]"`.
+- [ ] For major releases, prepare content for social media highlighting new features.
 
-Wait for the release candidate to be cut:
+_Wait for the release candidate to be created._
 
-## After release candidate has been cut
```

Comment:
cut is the wording we usually use but I don't object to created, might be easier to understand for new contributors :)

Reply:
That was my thinking here.